Tools/tsan/suppressions_free_threading.txt

```diff
@@ -0,0 +1,51 @@
+# This file contains suppressions for the free-threaded build. It contains the
+# suppressions for the default build and additional suppressions needed only in
+# the free-threaded build.
+#
+# reference: https://github.com/google/sanitizers/wiki/ThreadSanitizerSuppressions
+
+## Default build suppresssions
+
+race:get_allocator_unlocked
+race:set_allocator_unlocked
+race:mi_heap_visit_pages
+race:_mi_heap_delayed_free_partial
```

Comment:
If we're categorizing them, the mimalloc suppressions are probably "free-threading specific".

Reply:
Are these known by the mimalloc maintainers?

Reply:
They are most likely issues in how we use mimalloc in the free-threading build rather than issues upstream.